Update elastic-san-regions.md (#124222)
Azure/azure-content: includes/elastic-san-regions.md

- Norway East - LRS
- UAE North - LRS
- India Central - LRS

> Elastic SAN also supports the following regions without Availability Zonal support: 

Reviewer: ```suggestion
Elastic SAN is also available in the following regions, but without Availability Zone support: 
```

- Canada East - LRS
- North Central US - LRS
- Japan West - LRS

> In order to enable these regions, the following feature flag will need to be registered: 

Reviewer: ```suggestion
To enable these regions, run the following command to register the necessary feature flag: 
```
